.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: ["*"]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust_version: [stable, "1.34.0"]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set up Rust toolchain
+      run: |
+        rustup default ${{ matrix.rust_version }}
+        rustup component add clippy rustfmt
+
+    - name: Build and test (default features)
```

Comment:
I don't expect this library to ever have any default features, so this is likely unnecessary

Reply:
Bleh forgot about std, fine keep it 😛